headerCheck/mock/marshalizerMock.go

```diff
@@ -0,0 +1,53 @@
+package mock
```

Comment:
please move this in the testsCommon package

Valid for all mocks from this package

Reply:
moved all mocks to testsCommon